plugins/frontity-headtags/includes/hooks/class-frontity-headtags-post-type-hooks.php

```diff
@@ -76,14 +76,25 @@ public function register_admin_hooks() {
 	public function get_post_headtags( $post ) {
 		// Get post type and id.
 		$post_type = $post['type'];
+		$post_slug = $post['slug'];
 		$post_id   = $post['id'];
 
 		// Use them to create $key and $query vars.
-		$key   = "{$post_type}_{$post_id}";
-		$query = array(
-			'post_type' => $post_type,
-			'p'         => $post_id,
-		);
+		$key = "{$post_type}_{$post_id}";
+
+		// Initialize an empty query.
+		$query = array();
+
+		if ( 'page' === $post_type ) {
+			// Add page attributes if the query is for a page.
+			$query['page']     = '';
+			$query['pagename'] = $post_slug;
```

Comment:
I guess this is the thing that fixed the title on pages, right?

Reply:
Yeah, it's like the regular way to build queries for pages, and AIO SEO depends on that.